live/liveview.go

```diff
@@ -276,29 +281,41 @@ func (s *socket) serve(ctx context.Context) {
 		select {
 		case info := <-s.info:
 			r, err = s.handleInfo(ctx, info)
-			res = append(res, r)
+			if r != nil {
```

Comment:
`if err == nil` would be a safer check. same for similar lines below.

Reply:
went back and forth here.  originally checked error :)